src/controller/cve-id.controller/index.js

```diff
@@ -18,8 +18,8 @@ router.get('/cve-id',
         <h2>Access Control</h2>
         <p>All registered users can access this endpoint</p>
         <h2>Expected Behavior</h2>
-        <p><b>Regular, CNA & Admin Users:</b> Retrieves filtered CVE-IDs owned by the user's organization</p>
-        <p><b>Secretariat:</b> Retrieves filtered CVE-IDs owned by any organization</p>
+        <p><b>Regular, CNA & Admin Users:</b> Retrieves filtered CVE IDs owned by the user's organization</p>
+        <p><b>Secretariat:</b> Retrieves filtered CVE IDs owned by any organization</p>
   #swagger.parameters['$ref'] = [
     '#/components/parameters/cveIdGetFilteredState',
```

Comment:
for the description of state, it would be more clear if "CVE state" is changed to "CVE ID state" or, maybe best to just remove the word CVE. Currently it is confusing because CVE by itself refers to a CVE ID/CVE Record combination, but not all CVE IDs have records